espnet/tts/pytorch_backend/tts.py

```diff
@@ -268,6 +268,9 @@ def train(args):
         optimizer = torch.optim.Adam(
             model.parameters(), args.lr, eps=args.eps,
             weight_decay=args.weight_decay)
+    elif args.opt == 'noam':
```

Comment:
I think it's better to move this optimizer to the general place in the later PR.
This optimizer should be used even for RNN.

Reply:
maybe we should create OptimizerInterface like fairseq that wraps pytorch/chainer optimizers and has ability to register cmd args.